stonesoup/movable/action/move_position_action.py

```diff
@@ -0,0 +1,89 @@
+import numpy as np
+from abc import abstractmethod
+from typing import Iterator, Sequence
+
+from ...base import Property
+from ...sensor.action import ActionGenerator, Action
+from ...types.state import StateVector
+
+
+class MovePositionAction(Action):
+    """This is the base class for an action that changes the
+    position of a platform or sensor."""
+
+    def act(self, current_time, timestamp, init_value):
+        return self.target_value
+
+
+class GridActionGenerator(ActionGenerator):
+    """This is the base class for generators that generate actions in a grid like fashion."""
```

Comment:
Might be worth specifying in documentation that they generate a MovePositionAction, same for the generator below